src/lib.rs

```diff
@@ -43,6 +43,23 @@ impl MongodbSessionStore {
     /// ```
     pub async fn connect(uri: &str, db: &str, coll_name: &str) -> mongodb::error::Result<Self> {
         let client = Client::with_uri_str(uri).await?;
+        let create_index = doc! {
+            "createIndexes": coll_name,
+            "indexes": [
+                {
+                    "key" : {"expireAt": 1 },
```

Comment:
When I was testing with the mongo cli `db.coll_session.createIndex( { created": 1 } , { expireAfterSeconds: 300 } )` threw errors when I tried to replay it. Will this have the same issue?